### DISCUSSION TITLE
백엔드 서버 도커로 띄운 후에도 영수증페이지에서 계속 더미 데이터가 보이는 원인

### DISCUSSION THREAD

**Discussion by gygy7151** [EasyAndBeauty/DailyReceipt-FE-WEB#197] [Q&A]
<img width="1420" alt="image" src="https://user-images.githubusercontent.com/55748886/202977734-8e03638f-8945-4f51-9d81-60591a796667.png">

이렇게 성공적으로 도커 띄운 후
투두 항목을 작성하고
영수증 페이지에 들어가 확인해보니
아래와 같이 더미데이터가 나옵니다.
<img width="675" alt="image" src="https://user-images.githubusercontent.com/55748886/202977990-3fa9d539-ffdd-493e-bae2-42be7f182234.png">

제 로컬에 띄운 백엔드 서버랑 통신이 안되고 있어서 이런 현상이 발생하는 건가요?
이유가 무엇인지 너무너무 궁금해서 질문 남깁니다.😹

그리고 영수증 페이지에서 각 항목별 누적시간 단위는 00분:00초가 맞을까요?
헷갈려서 질문 올립니다.

**Comment by mdgarden** [comment]
- 더미데이터가 나오는 이유

왜냐하면 더미데이터를 연결시켜놨기 때문입니다 😂
마이페이지에서는 pin된 영수증만 불러와야하는데, 그 API 연결작업은 레몬이랑 할 예정입니다.
저희는 이번에 더미데이터로 이동하는 것까지만 하면 될 것 같아요.

- 시간 카운팅

> 그리고 https://www.daily-receipt.com/receipt 와 http://localhost:3000/receipt 모두
> 시간 카운팅 반영이 비정상적으로 너무 크게 잡히고 있었습니다.
> 40초만 지났는데 168:1로 누적시간이 표시되는 것을 보고 의아하더라구요

제 환경에서는 재현이 안되는데, 구체적인 재현방법 이슈로 등록 부탁드립니다.

**Reply by gygy7151** [reply]
바쁘신 와중에 답변 고맙습니다 마당!!💚
구체적인 재현 방법 #198 등록 완료했습니당~
